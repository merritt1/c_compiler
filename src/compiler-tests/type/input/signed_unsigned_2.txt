unsigned char a;
signed int c;
int main(void) {
a + c;
c + a;
}
