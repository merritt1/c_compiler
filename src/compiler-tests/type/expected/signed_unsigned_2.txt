=============== PARSE TREE ===============
unsigned char (a);
signed int (c);
signed int (main(void)) {
((signed int) a+c);
(c+(signed int) a);
}
================= SYMBOLS ================
Symbol Table ID: 0
  a -> unsigned char
  c -> signed int
  main -> function(signed int, [void])

